### DISCUSSION TITLE
Bibliography and citations / ref in markdown

### DISCUSSION THREAD

**Discussion by reyman** [srid/emanote#124] [Ideas]
Hi @srid , thanks for you wonderfull work on emanote.

I open a discussion because i'm surprised that bibliography doesn't appear in the documentation of emanote.
Pandoc with pandoc and option --citeproc (and also the fork of pandoc-citeproc without pandoc dependency [citeproc](https://hackage.haskell.org/package/citeproc)  is a reference tool for generating bibliography.

Do you plan to add the possibility to cite/ref in markdown (using [pandoc standard](https://pandoc.org/MANUAL.html#extension-citations)) with emanote ?  

Best regards,

**Comment by srid** [comment]
Hi! Yes, I definitely think this is a good feature to be added to Emanote. 

I don't use it myself so far, but I can see it being useful even in my own Zettelkastens. Emanote already uses pandoc standard, so what's left is to come up with a design while understanding how pandoc citeproc stuff works. 

The corresponding neuron feature request has some info: https://github.com/srid/neuron/issues/134

Note that a 1.0 release (see README) is my current priority.

**Comment by j-steinbach** [comment]
Probably should be its own discussion, but (anecdotally speaking), many/most or at least a sizeable chunk of people that are interested in bibliography also run Zotero+BBT. The latter has something called [Pull Export](https://retorque.re/zotero-better-bibtex/exporting/pull/), which allows fetching of the [bibliography](https://retorque.re/zotero-better-bibtex/citing/cayw/#diy) programmatically. This might be useful for e.g. queries or references look-up. 

On the other hand: Nothing speaks against just having a .bibtex file in the root folder.


(Also :+1:  for citations/bibliography)
